pkg/broker/handler/providers.go

```diff
@@ -45,7 +45,7 @@ var (
 				MaxConnsPerHost:     500,
 				IdleConnTimeout:     30 * time.Second,
 			},
-			Propagation: &tracecontext.HTTPFormat{},
+			Propagation: tracecontextb3.TraceContextEgress,
```

Comment:
Is this the same argument as above? Namely that because this is an `http.Client`, it is inherently an egress only thing, and therefore doesn't need this new format?